### DISCUSSION TITLE
Consider add ability to exact search

### DISCUSSION THREAD

**Discussion by Dentrax** [community/community#9715] [Other Feature Feedback, Questions, & Ideas] [answered]
It would be a great feature. 🚀 

![Screen Shot 2022-01-06 at 21 27 07](https://user-images.githubusercontent.com/16493751/148432305-9c11e310-ab45-4587-84da-f6f683ac4ceb.png)

**Comment by fiffeek** [comment, marked as answer]
It already is, just use double quotes over the prompt: `"if err != nil"`

**Reply by Dentrax** [reply]
Thanks, totally missed this. My bad. 🙏 

![Screen Shot 2022-01-06 at 21 52 17](https://user-images.githubusercontent.com/16493751/148435465-031ccf62-2e28-4929-949f-74d7b9e6bd8e.png)
